tests/integration-service/integration.go

```diff
@@ -98,6 +99,17 @@ var _ = framework.IntegrationServiceSuiteDescribe("Integration Service E2E tests
 					return nil
 				}, timeout, interval).Should(Succeed())
 			})
+
+			It("verifies that the finalizer has been removed", func() {
+				timeout := "60s"
+				interval := "1s"
+				Eventually(func() error {
+					if !utils.Contains(pipelineRun.Finalizers, "build.appstudio.openshift.io/pipelinerun") {
```

Comment:
If I'm not reading this condition wrong, it seems like this checks if the `build.appstudio.openshift.io/pipelinerun` is _not_ present on the pipelineRun, but then the Error that is raised says the opposite, that the build pipelineRun %s/%s still contains the finalizer.

Reply:
You're right, I created a new quickcluster and am running tests now.  It's a slow process.  Don't want you to think I didn't see this.

Reply:
Yup, looks good now, thanks.